DancingGoat/Views/Shared/_Layout.cshtml

```diff
@@ -15,25 +15,25 @@
                     <nav role="navigation">
                         <ul>
                             <li>
-                                @Html.ActionLink(Localizer["Home"], "Index", "Home")
```

Comment:
I can't see how `@Html.ActionLink` could have ever worked since the localization library only catches the tag helper.
https://github.com/Kentico/aspnetcore-localizedrouting/blob/master/src/Kentico.AspNetCore.LocalizedRouting/TagHelpers/LocalizedRoutingAnchorTagHelper.cs